bridge/MixerManager.cpp

```diff
@@ -56,6 +56,14 @@ MixerManager::~MixerManager()
 
 Mixer* MixerManager::create()
 {
+    return create(_config.defaultLastN);
+}
+
+Mixer* MixerManager::create(uint32_t lastN)
```

Comment:
Good, I wasn't sure how we would handle lastN from each person joining. Now it will be the initiator that controls this.